hugo/layouts/report.html

```diff
@@ -715,38 +761,33 @@ <h5>${comp.is_subject_pin_sale ? (cardData.is_multicard ? "Subject card" : "Subj
     }
 
     // Initialize the characteristic comparison table
-    function initializeTable(tableId) {
-      // Extract and store frozen rows
-      const $table = $(`#${tableId}`);
-      const $frozenRows = $table.find("tr.frozen-row").clone(); // Store clones
-
-      // Remove original frozen rows from DOM so they don't get indexed
-      $table.find("tr.frozen-row").remove();
-
+    function initializeTable(cardNum, cardIndex) {
+      const $table = $(`#comp-table-${cardNum}`);
       const dt = $table.DataTable({
-        paging: true,
-        searching: true,
-        info: false,
-        pageLength: 7,
-        ordering: true,
-        order: [], // Disable default ordering
-        lengthChange: false,
-        language: {
-          search: "Filter characteristics:"
+        paging: false,  // Disable pagination to display all chars at once
+        scrollX: true,  // Allow the table to scroll horizontally on narrow screens
+        scrollY: "500px",  // Force the table to scroll vertically with a fixed height
+        scrollCollapse: true,  // Collapse the table size below the fixed height when there are few rows
+        fixedColumns: true,  // Freeze the leftmost column (chars) during horizontal scroll
+        fixedHeader: {
+          header: true,  // Make the header row sticky during scroll
+          footer: true  // Make the footer sticky during scroll
+        },
+        searching: true,  // Enable a search box to filter rows
+        info: true,  // Enable info text (defined under the `language` key below)
+        ordering: false,  // Disable sorting by column
+        language: {  // Custom text
+          search: "<i class='bi bi-search' aria-hidden='true'></i> <b>Search</b> characteristics:",  // Search input label
+          info: "<span class='d-lg-none'>Scroll right to see more sales <i class='bi bi-arrow-right' aria-hidden='true'></i></span>",
+          infoFiltered: "",  // Disable searched row count text
+          infoEmpty: "",  // Disable text that shows when no rows match search
+        },
+        layout: {
+          topStart: "search",  // Top left: Search box
+          topEnd: "info",  // Top right: Horizontal scroll prompt
+          bottomStart: null,  // Disable bottom text, since we use a frozen footer
```

Comment:
This is the meat of this PR! I've tried to document all of the attributes we're using as clearly as possible, but I encourage referring back to the [DataTables documentation](https://datatables.net/reference/option/) for anything that is unclear.